src/content/quickstart/openshiftgn/_index.md

```diff
@@ -84,6 +84,9 @@ subctl join --kubecontext west --kubeconfig ./merged_kubeconfig broker-info.subm
 subctl join --kubecontext east --kubeconfig ./merged_kubeconfig broker-info.subm --clusterid east --broker-cluster-context west
 ```
 
+This will auto-assign Global CIDRs to the clusters. ```--globalnet-cidr``` flag can be used if Globalnet CIDRs need to be externally specified by the user.
+subctl will check that the specified CIDRs are not overlapping and other sanity checks before deploying the clusters.
```

Comment:
```suggestion
subctl will check that the specified CIDRs do not overlap across clusters before joining the clusters.
```

Reply:
Please apply the changes before marking the conversation as Resolved. As @tpantelis pointed out, we are not deploying the cluster, we are "joining" the clusters.

Also, like I commented earlier here- https://github.com/submariner-io/submariner-website/pull/93#pullrequestreview-405697504, the quickstart guide should be short and succinct. But this opinion could be subjective. If others are fine including this message here, it's Okay to me. 

We can also do the following, remove this description from quickstart and explicitly include **--globanet-cidr 169.254.0.0/19 for cluster-a** and **--globanet-cidr 169.254.32.0/19 for cluster-b** as part of join command above. This will be a solution to some of the race conditions we are seeing until the [issue](https://github.com/submariner-io/submariner-operator/issues/299) is resolved.

Reply:
> Please apply the changes before marking the conversation as Resolved. As @tpantelis pointed out, we are not deploying the cluster, we are "joining" the clusters.
> 
I missed the joining part. As can be seen other comments which need discussion are still open and not resolved.

> Also, like I commented earlier here- [#93 (review)](https://github.com/submariner-io/submariner-website/pull/93#pullrequestreview-405697504), the quickstart guide should be short and succinct. But this opinion could be subjective. If others are fine including this message here, it's Okay to me.
> 
> We can also do the following, remove this description from quickstart and explicitly include **--globanet-cidr 169.254.0.0/19 for cluster-a** and **--globanet-cidr 169.254.32.0/19 for cluster-b** as part of join command above. This will be a solution to some of the race conditions we are seeing until the [issue](https://github.com/submariner-io/submariner-operator/issues/299) is resolved.

I like adding the flags as part of join command. And a short description of it is anyways added to the architecture section.